Rivet/Functions/Set-DefaultConstraintName.ps1

```diff
@@ -0,0 +1,42 @@
+
+function Set-DefaultConstraintName
+{
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory,ValueFromPipeline)]
+        [Rivet.Column]$Column,
+
+        [Parameter(Mandatory)]
+        [String]$SchemaName,
+
+        [Parameter(Mandatory)]
+        [String]$TableName
+    )
+
+    process
+    {
+        Set-StrictMode -Version 'Latest'
+
+        if( -not $Column.DefaultExpression )
+        {
+            $Column.DefaultConstraintName = $null
+            return $Column
+        }
+
+        if( $Column.DefaultExpression -and $Column.DefaultConstraintName )

```

Comment:
Condition cascading can be simplified.
- $Column.DefaultExpression always false here.
- $Column.DefaultConstraintName pairing can be changed to:

     if (-not $Column.DefaultConstraintName) { /* set */ } ..